common/source/docs/common-object-avoidance-landing-page.rst

```diff
@@ -4,8 +4,9 @@
 Object Avoidance
 ================
 
-Ardupilot supports many kinds of object avoidance in vehicles. Airborne Vehicles (ADSB), Object Avoidance (Object/Ground/Ceiling), and Geo-Fences (Beacons, Exclusion zones, Fences).
-Supported types vary with vehicle. Some kinds of avoidance require external hardware, such as ADSB receivers or  Rangefinders.
+Ardupilot supports several kinds of object avoidance in vehicles. Avoidance of Airborne Vehicles (ADSB) and Object Avoidance (Object/Ground/Ceiling).
```

Comment:
a small thing but we can probably remove "in vehicles".  We don't actually support avoiding the ground although I see this is an existing issue with the docs.

Reply:
will do